assignment 6/promisesassignment.js

```diff
@@ -0,0 +1,53 @@
+var fs = require("fs");
+
+var firstMethod = function() {
+  var promise = new Promise(function(resolve, reject) {
+    fs.readFile("file1.txt", function(err, data) {
+      if (err) {
+        reject(err);
+      }
+      console.log("first method completed");
+      resolve(data);
+    });
+  });
+  return promise;
+};
+
+var secondMethod = function(data) {
+  var promise = new Promise(function(resolve, reject) {
```

Comment:
@abansal26 Directly return promise, no need to use extra variable 'promise' here.

Reply:
Changes as suggested by @sakodiya commited. Commit ID - de7738060960035e58c09b39005e3c5b761817c6